tests/unit/test_pos.py

```diff
@@ -35,8 +41,13 @@ def scrape_pos_page_body():
 
 
 @pytest.fixture
```

Comment:
you can make this fixture 'session', so it does not load it from disk every time.